packages/zarathustra/connections/openai_api/connection.py

```diff
@@ -49,7 +49,7 @@
 CONNECTION_ID = PublicId.from_str("zarathustra/openai_api:0.1.0")
 _default_logger = logging.getLogger("aea.packages.zarathustra.connections.openai_api")
 
-LLM_RESPONSE_TIMEOUT = 10
+LLM_RESPONSE_TIMEOUT = 30
```

Comment:
had to increase, otherwise new user summaries will timeout since there's a sufficient amount of data there